### PR TITLE
feat(quick-trace): Use trace-meta for counts

### PR DESCRIPTION
**Pull request by Zylphrex**
The `events-meta` endpoint checks for the `global-views` flag. This is
problematic because we want the counts to be correct even when the
`global-views` flag is off. This switches it from the `events-meta` enpoint to
the `trace-meta` endpoint.

### CODE REVIEW

File: src/sentry/static/sentry/app/utils/performance/quickTrace/quickTraceQuery.tsx [hunk 1]
```diff
@@ -54,7 +64,10 @@ export default function QuickTraceQuery({children, event, ...props}: QueryProps)
                   const trace = flattenRelevantPaths(event, subtrace);
                   return children({
                     ...traceFullResults,
+                    isLoading: traceFullResults.isLoading || metaResults.isLoading,
+                    error: traceFullResults.error || metaResults.error,
```
Comment (wmak) [inline src/sentry/static/sentry/app/utils/performance/quickTrace/quickTraceQuery.tsx:68]: Doesn't grouping trace + meta mean that if meta broke for whatever reason the entire page would appear to error when we could've loaded the trace still?

Comment (Zylphrex) [reply src/sentry/static/sentry/app/utils/performance/quickTrace/quickTraceQuery.tsx:68]: Hm... So the impact here is just for quick trace (trace view is not affected here), but that is a good point. I can split it out to get around this problem

Comment (Zylphrex) [reply src/sentry/static/sentry/app/utils/performance/quickTrace/quickTraceQuery.tsx]: resolved in 6540a2c

**Review by wmak — APPROVED** [review]
LGTM

**Pull request merged**